simul/simul.go

```diff
@@ -260,3 +304,32 @@ func getStartStop(rcs int) (int, int) {
 	dbg.Lvl2("Range is", start, ":", stop)
 	return start, stop
 }
+
+// getRunWait returns either the command-line value or the value from the runconfig
+// file
+func getRunWait(rc platform.RunConfig) int {
```

Comment:
I'm OK with that for the moment, but at some point later we should start extending this `RunConfig` struct. Having something like:
```
// Returns the value of the field designated by `name` or the `default` value if the fields does not exists.
func (rc *RunConfig) GetInt(name string, default int) int {
    ...
}

... 
rc.GetInt("runwait",100)
````

The same for `string`, `float64` etc